### PR TITLE
@/codeforafrica/components/input

### LINKED ISSUE #150: @/codeforafrica/components/Input
Implement the (search) Input component per [designs](https://www.figma.com/file/MbCygAaNhIjgJOHrQZpZ2U/CfA---CfA-site?node-id=5138%3A3775) and add it to the respective pages/components.

**NOTE**: This is just a UI implementation. Back-end integration will be done in a separate story.

### Screenshot

Image

### PR DESCRIPTION
**Pull request by kilemensi**
## Description

Resolves #150 .

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshots

S

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: apps/codeforafrica/src/components/FeaturedProjects/FeaturedProjects.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import ChoiceChip from "@/codeforafrica/components/ChoiceChip";
 import ChoiceChipGroup from "@/codeforafrica/components/ChoiceChipGroup";
 import ProjectTileList from "@/codeforafrica/components/ProjectTileList";
 
-const DEFAULT_CATEGORY = "Projects";
+const DEFAULT_CATEGORY = "Products";
```
Comment (esirK) [inline apps/codeforafrica/src/components/FeaturedProjects/FeaturedProjects.js:8]: yeah, I noticed this too; We had a `Projects` category which is confusing.

File: apps/codeforafrica/src/components/Input/Input.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      borderColor: "#D0CBCB",
+      top: 0,
+    },
+    "& fieldset legend": {
+      display: "none",
+    },
+  },
+}));
+
+const Input = React.forwardRef(function Input(props, ref) {
```
Comment (esirK) [inline apps/codeforafrica/src/components/Input/Input.js:54]: Just out of curiosity, I see the need for `forwardRef` on this Input but I've also seen it in almost all other components that aren't really Inputs; Why is this?

File: apps/codeforafrica/src/components/Input/Input.test.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { createRender } from "@commons-ui/testing-library";
+import React from "react";
+
+import Input from "./Input";
```
Comment (esirK) [inline apps/codeforafrica/src/components/Input/Input.test.js:4]: This works but for consistency, wouldn't it be better to always import from `@/codeforafrica/...`?

File: apps/codeforafrica/src/components/Input/Input.js (hunk 2 above)
Comment (kilemensi) [reply apps/codeforafrica/src/components/Input/Input.js:54]: It's not strictly required @esirK  but recommended if you're developing reusable components. More on [info](https://reactjs.org/docs/forwarding-refs.html)

**Comment by esirK** [comment]
@kilemensi  I think it's per design but the Search text and the cancel button seems not to align with the search Icon
Screenshot 2022-07-01 at 15 10 45

Screenshot 2022-07-01 at 15 11 03

File: apps/codeforafrica/src/components/Input/Input.test.js (hunk 3 above)
Comment (kilemensi) [reply apps/codeforafrica/src/components/Input/Input.test.js:4]: When you're importing from same folder, this is still the recommended approach. Issues come when you're importing from other folders relative a give file e.g. `import X from "../../components/Y";`. This is where aliasing shine @esirK .

**Comment by kilemensi** [comment]
> @kilemensi I think it's per de....
> 

Yes @esirK, it's per [designs](https://www.figma.com/file/MbCygAaNhIjgJOHrQZpZ2U/CfA---CfA-site?node-id=5138%3A3776)... The icon is perfectly centred (equal top & bottom padding) but text input is not (more top padding than bottom). The cancel icon takes text input alignment (It's added by Chrome/Safari? I don't see it in Firefox).

#### Icon
Screenshot from 2022-07-01 15-14-58

#### Text input

Screenshot from 2022-07-01 15-14-38

**Comment by esirK** [comment]
> > @kilemensi I think it's per de....
> > 
> 
> Yes @esirK, it's per [designs](https://www.figma.com/file/MbCygAaNhIjgJOHrQZpZ2U/CfA---CfA-site?node-id=5138%3A3776)... The icon is perfectly centred (equal top & bottom padding) but text input is not (more top padding than bottom). The cancel icon takes text input alignment (It's added by Chrome/Safari? I don't see it in Firefox).
> 
> #### Icon
> Screenshot from 2022-07-01 15-14-58
> 
> #### Text input
> 
> Screenshot from 2022-07-01 15-14-38
> 
> 

Currently on Chrome

**Pull request merged**